src/openseadragon.js

```diff
@@ -187,6 +187,9 @@
   * @property {Number} [opacity=1]
   *     Default opacity of the tiled images (1=opaque, 0=transparent)
   *
+  * @property {Boolean} [preload=false]
```

Comment:
I wasn't expecting this to be a global flag, but I guess it makes sense since we already have `opacity` as a global flag. That said, I don't know that we should be encouraging people to turn it on in a blanket way... it's probably more effective when used strategically. Just thinking out loud here...